src/marqo/tensor_search/tensor_search.py

```diff
@@ -1595,7 +1585,19 @@ def get_query_vectors_from_jobs(
             ]
             # TODO how doe we ensure order?
             weighted_vectors = [np.asarray(vec) * weight for vec, weight, content in vectorised_ordered_queries]
-            merged_vector = np.mean(weighted_vectors, axis=0)
+            
+            custom_tensors = q.get_context_tensor() 
+            if custom_tensors is not None:
```

Comment:
custom tensor support for bulk_search (and search to use this method).

Reply:
should we call this var `context_tensors` to be consistent?